app/Libraries/Search/ScoreSearch.php

```diff
@@ -79,7 +79,9 @@ private function addModsFilter(BoolQuery $query): void
         }
 
         $modsHelper = app('mods');
-        $allMods = new Set(array_keys($modsHelper->mods[$this->params->rulesetId]));
+        $allMods = new Set(isset($this->params->rulesetId)
+            ? array_keys($modsHelper->mods[$this->params->rulesetId])
+            : array_merge(...array_map('array_keys', array_values($modsHelper->mods))));
```

Comment:
seems like something that can be memoized if only to give it a name instead of `array_merge(...array_map....array.....` (or have an `allModsForRuleset` function or something)